modules/pel/peltool/peltool.py

```diff
@@ -267,6 +319,8 @@ def main():
                         help='Used with -d, only look for files with this extension (e.g. ".pel")')
     parser.add_argument('-x', '--delete', dest='delete', action='store_true',
                         help='Delete original file after parsing')
+    parser.add_argument('-l', '--list', nargs='?', const="/var/lib/phosphor-logging/extensions/pels/logs/", 
```

Comment:
For now, I don't want to expose these arguments to people running peltool when they aren't on the BMC.  I think we could get away which checking if we're on the BMC by looking to see if a certain directory is present, like either /usr/share/phosphor-logging or /var/lib/phosphor-logging.

After everything is in and working, and then we can reconsider if we'd want to show some or all of them.

Reply:
Yeah, we can use this option even if we are not in a BMC. 

With the current code, if a user doesn't enter anything with -l, it will take default location as mentioned in const. 
However, if the user adds a directory after -l, then that directory will be stored in args.list.

For example:
```bash
python3 -m pel.peltool.peltool -l
```
Here args.list will store "/var/lib/phosphor-logging/extensions/pels/logs/"
```bash 
python3 -m pel.peltool.peltool -l testfolder/ 
```
Here args.list will store "testfolder/"

Reply:
Got it!

Reply:
I should add, the same goes with all of these new options.

Reply:
If we were going to show -l on non-BMC, it would need to detect that and require an argument.

Reply:
Resolved!
In a non-BMC it will simply ask the user for a folder path.